### PR TITLE
Milla_Login

### PR DESCRIPTION
**Pull request by LIUDNY**

### CODE REVIEW

File: src/test/resources/features/LogIn.feature [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  And I type password "123456"
+  When I hit button "Sign In"
+  And I wait for 4 seconds
+  When I click "Log Out"
+  Then confirmation window should appear
+  And I click "Log Out" button
+  Then return to "login"page
+
+    @LogIn2
+    Scenario: LogIn Page Register Now with valid credentials
```
Comment (gpl-git) [inline src/test/resources/features/LogIn.feature:17]: This test does not belong to this test set. Please remove

File: src/test/java/definitions/LogInStepdefs.java [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+    @Then("I see {string} page")
+    public void iSeePage(String page) {
+        getDriver().navigate().back();
+    }
+
+
+    @Then("password field should contain error message")
+    public void passwordFieldShouldContainErrorMessage() {
+        getDriver().findElement(By.xpath("//input[@formcontrolname='password']"));
+
```
Comment (gpl-git) [inline src/test/java/definitions/LogInStepdefs.java:140]: xpath //mat-error
assertThat(getDriver().findElement(By.xpath("//mat-error")).isDisplayed())isTrue();.

File: src/test/java/definitions/LogInStepdefs.java [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+    @Then("pop up window with error message text should show up")
+    public void popUpWindowWithErrorMessageTextShouldShowUp() {
+        getDriver().findElement(By.xpath("//div[@class='cdk-visually-hidden']"));
+    }
+
+    @Then("password should be masked")
+    public void passwordShouldBeMasked() {
+        getDriver().findElement(By.xpath("//input[@placeholder='Password *']"));
+
```
Comment (gpl-git) [inline src/test/java/definitions/LogInStepdefs.java:157]: assertThat(getDriver().findElement(By.xpath("//input[@formcontrolname='password']")).getAttribute("type").equals("password")).isTrue();

File: src/test/resources/features/LogIn.feature [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+    @LogIn11
+                  Scenario: Trailing spaces are not allowed
+                    Given I navigate to "login" page
+                    And I wait for 4 seconds
+                    When I enter "lmislinski@nbobd.com" into email field
+                    And type enter
+                    Then email field should contain error message
+
+    @LogIn12
```
Comment (gpl-git) [inline src/test/resources/features/LogIn.feature:110]: Delete scenario number 12

File: src/test/resources/features/LogIn.feature (hunk 1 above)
Comment (LIUDNY) [reply src/test/resources/features/LogIn.feature:17]: Done

**Comment by LIUDNY** [comment]
Done

**Pull request closed without merging**